test/parallel/test-eslint-lowercase-name-for-primitive.js

```diff
@@ -7,45 +7,33 @@ common.skipIfEslintMissing();
 const RuleTester = require('../../tools/node_modules/eslint').RuleTester;
 const rule = require('../../tools/eslint-rules/lowercase-name-for-primitive');
 
-const valid = [
-  'string',
-  'number',
-  'boolean',
-  'null',
-  'undefined'
-];
-
 new RuleTester().run('lowercase-name-for-primitive', rule, {
   valid: [
     'new errors.TypeError("ERR_INVALID_ARG_TYPE", "a", ["string", "number"])',
-    ...valid.map((name) =>
-      `new errors.TypeError("ERR_INVALID_ARG_TYPE", "name", "${name}")`
-    )
+    'new errors.TypeError("ERR_INVALID_ARG_TYPE", "name", "string")',
+    'new errors.TypeError("ERR_INVALID_ARG_TYPE", "name", "number")',
+    'new errors.TypeError("ERR_INVALID_ARG_TYPE", "name", "boolean")',
+    'new errors.TypeError("ERR_INVALID_ARG_TYPE", "name", "null")',
+    'new errors.TypeError("ERR_INVALID_ARG_TYPE", "name", "undefined")',
```

Comment:
Are we moving to trailing commas?

Reply:
> Are we moving to trailing commas?

@lpinca We use both styles in the code base. I would never block or even nit anyone else's PRs about it, but yes, I try to add trailing-commas when the opportunity presents itself.

FWIW, right now on `master`, if we change the `comma-dangle` rule to this&hellip;:

```js
    'comma-dangle': ['error', {
      arrays: 'always-multiline',
      objects: 'only-multiline',
    }],
```

&hellip;we get 689 lines to change across 367 files. That's not something I'd want to do in a single `comma-dangle` PR but also something that seems achievable over enough time. (We've done it for other rules.)

In https://github.com/nodejs/node/pull/19133, it seemed agreed that this approach (change when you can if you're already in there for other reasons) was a reasonable approach.

Reply:
I'm not suggesting to enable the rule or blocking this for such a nuance, I was only only interested to know if any decision was made about that. In a recent PR discussion (I'll try to find it and link here) some collaborators, me included, were not very happy with trailing commas.

Reply:
Refs: https://github.com/nodejs/node/pull/20703#issuecomment-388642420

Reply:
@lpinca I suspect you mean the voting on this comment: https://github.com/nodejs/node/pull/20703#issuecomment-388642420

You, tniessen, and cjihrig went for "no trailing comma". 

A little later on, bnoordhuis, targos, and mmarchini seemed to be in favor of it (definitely bnoordhuis, and inferring the other two from the GitHub reactions on https://github.com/nodejs/node/pull/20703#issuecomment-388754052).

Dangling-comma vs. no-dangling-comma may finally be impossible for the project to resolve unlike the venerable tabs vs. spaces and semicolons vs. no-semicolons.

Reply:
> I was only only interested to know if any decision was made about that. 

In that case: Definitely not. No decision has been made.